lib/bugsnag/integrations/delayed_job.rb

```diff
@@ -10,11 +10,22 @@
 module Delayed
   module Plugins
     class Bugsnag < ::Delayed::Plugin
+      ##
+      # DelayedJob doesn't have an easy way to fetch its version, but we can use
+      # Gem.loaded_specs to get the version instead
+      def self.delayed_job_version
+        ::Gem.loaded_specs['delayed_job'].version.to_s
+      rescue StandardError
+        nil
```

Comment:
This is to make Rubocop happy — if we don't explicitly put a `nil` here it gets upset about suppressing exceptions

Reply:
This should get a little comment or something - I could easily see deleting the `nil` in the future because it doesn't normally matter.